src/components/routes/Listing/Listing/ListingInformation.tsx

```diff
@@ -120,11 +122,30 @@ const ListingInformation = ({ listing, host }: Props) => {
               // TODO: When page is refreshed, supportEmail is null due to firebase user query and our listing query running in parallel.
```

Comment:
can we remove this comment now that we don't use supportEmail?

Reply:
removed